pkg/operator/resourcesynccontroller/resourcesynccontroller.go

```diff
@@ -32,18 +32,6 @@ func NewResourceSyncController(
 	); err != nil {
 		return nil, err
 	}
-	if err := resourceSyncController.SyncConfigMap(
```

Comment:
the aggregator-client-ca is logically a union of other certs.  This was a mistake from back when.